app/liquid/liquid_file_system.rb

```diff
@@ -1,5 +1,16 @@
 class LiquidFileSystem
   def read_template_file(title)
-    LiquidPartial.where(title: title).first.content
+    view = LiquidPartial.find_by(title: title)
+
+    if view
+      return view.content
+    else
+      partials = Dir.glob("#{Rails.root}/app/views/plugins/**/_#{title}.liquid")
+      if partials.any?
+        File.read partials.first
+      else
+        "Partial #{title} was not found"
+      end
+    end
```

Comment:
Customised liquid's filesystem to search our `views/plugins` directory for partial when not found in the database.

It's ugly, and needs refactoring, but ok for MVP